ravenframework/SupervisedLearning/DMD/DynamicModeDecompositionControl.py

```diff
@@ -199,6 +236,43 @@ def _handleInput(self, paramInput):
       if str(self.parametersIDs[i]).endswith('_init'):
         self.parametersIDs.remove(self.parametersIDs[i])
 
+  def initializeModel(self, dmdParams):
+    """
+      Method to initialize the surrogate model with a dmdParams dictionary
+      @ In, dmdParams, dict, the dictionary containin the parameters/settings to instanciate the model
+      @ Out, None
+    """
+    pass
```

Comment:
remove?

Reply:
This is overloaded to avoid conflicts with the DynamicModeDecompositionControl (that is used by FARM external plugin). Maybe once this is merged, ANL can be informed to check how the DynamicModeDecompositionControl (now also available in PyDMD) can be recast.